src/Java.Interop/Java.Interop/JniRuntime.cs

```diff
@@ -414,16 +414,12 @@ partial class JniRuntime {
 
 		public virtual void RaisePendingException (Exception pendingException)
 		{
+#if XA_INTEGRATION
```

Comment:
Why move this line? Don't we *always* want to throw `ArgumentNullException` if `pendingException` is `null`?

Reply:
It is checked in https://github.com/radekdoulik/java.interop/blob/b9ee7b1527fee332d9e1323790a7c6847984a3c2/src/Java.Interop/Java.Interop/JniEnvironment.Errors.cs#L32 which we call in `!XA_INTEGRATION` case.